README.md

```diff
@@ -1,48 +1,159 @@
+<a name="readme-top"></a>
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+<div align="left">
+</div>
 
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-## Getting Started
+Modify this file to match your project and remove sections that don't apply.
 
-This repository includes files with plain SQL that can be used to recreate a database:
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+After you're finished please remove all the comments and instructions!
+-->
 
+<div align="center">
 
-## Authors
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-👤 **Author1**
+  <h3><b>Microverse README Template</b></h3>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+</div>
 
-👤 **Author2**
+<!-- TABLE OF CONTENTS -->
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+# 📗 Table of Contents
 
-## 🤝 Contributing
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Countries metrics] <a name="about-project"></a>
+
+> **[Vet clinic dtabase]** A database to a veterinary clinic
+
+## 🛠 Built With <a name="built-with">PostgreSQL</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Loom video <a name="live-demo"></a>
+
+> A video Explaining The app
+- [Loom Video](https://www.loom.com/share/9a2411e8409d42c89e45e458f1a4ce84)
```

Comment:
- All sections that don't apply to this project, you are supposed to get rid of it. Please don't just copy/paste without going through the entire README template. This is not professional.